tests/unit/MDMMotionCurveTests.m

```diff
@@ -53,4 +53,43 @@ - (void)testSpringCurveDataWithMacro {
   XCTAssertEqualWithAccuracy(curve.data[MDMSpringMotionCurveDataIndexFriction], 0.3, 0.001);
 }
 
+- (void)testSystemModalMovementTimingCurveMatchesModalMovementTiming {
+  UIWindow *window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+  window.rootViewController = [[UIViewController alloc] init];
+  [window makeKeyAndVisible];
+
+  UIViewController *presentedViewController = [[UIViewController alloc] init];
+  XCTestExpectation *didComplete =
+      [[XCTestExpectation alloc] initWithDescription:@"Animation completed."];
```

Comment:
I believe the recommended way to create expectations is

```objectivec
XCTestExpectation *expect = [self expectationWithDescription:@"blah"];
```
[This only works with Xcode 7.2+ according to the docs](https://developer.apple.com/documentation/xctest/xctestcase/1500899-expectationwithdescription?language=objc). Does Motion-Interchange need to support earlier Xcode versions?

Reply:
Done.